app/Http/Controllers/Api/Webform/ExternalWebformController.php

```diff
@@ -2119,4 +2092,42 @@ protected function addTaskError(string $error)
         $this->taskErrors[] = $error;
     }
 
+    /**
+     * @return bool
+     */
+    private function createHoomDossier()
+    {
+        $cooperation = Cooperation::first();
+        $this->log("Aanmaken hoomdossier contact");
+        if (!$cooperation || empty($cooperation->hoom_link)) {
+            $this->log("Kan geen Hoomdossier aanmaken want er is bij cooperatie geen hoomdossier link gevonden.");
+        } else {
+            if (!$this->contact) {
+                $this->log("Kan geen Hoomdossier aanmaken want er is geen contact gevonden.");
```

Comment:
Dit kan netter, bijv.

if (!$this->contact) {
   return $this->log("Kan geen Hoomdossier aanmaken want er is geen contact gevonden.");
}

if ($this->contact && $this->contact->hoom_account_id) {
   return $this->log("Koppeling hoomdossier bestaat al.");
}

try { ....